Library/Formula/nload.rb

```diff
@@ -11,6 +11,11 @@ class Nload < Formula
   depends_on "autoconf" => :build
   depends_on "automake" => :build
 
+  # F2 Option Crash and Wrong adapter read, see http://sourceforge.net/p/nload/bugs/8/
+  patch :p0 do
+    url "http://sourceforge.net/p/nload/bugs/_discuss/thread/c9b68d8e/4a65/attachment/devreader-bsd.cpp.patch"
```

Comment:
Let's optimistically use `https://` here. It works, but redirects for now. The hope is one day SF will stop redirecting secure to plaintext.